packages/create-remix/templates/koa/server/index.js

```diff
@@ -0,0 +1,55 @@
+const path = require("path");
+const Koa = require("koa");
+const compress = require("koa-compress");
+const morgan = require("koa-morgan");
+const static = require("koa-static");
+const { createRequestHandler } = require("@remix-run/koa");
+
+const MODE = process.env.NODE_ENV;
+const BUILD_DIR = path.join(process.cwd(), "server/build");
+
+const app = new Koa();
+app.use(compress());
+
+// You may want to be more aggressive with this caching
+app.use(static("public", { maxage: 3600000 }));
+
+// Remix fingerprints its assets so we can cache forever
+app.use(
+  static("public/build", {
+    setHeaders: res => {
+      res.set("Cache-Control", true);
+    },
+    maxage: 31557600000
+  })
+);
```

Comment:
This will never get hit because the first `static` will greedily take anything in `public/build` too. See https://github.com/remix-run/remix/pull/678 where the same problem for the `express` package has been fixed.

Reply:
fixed, thank you.